examples/default/src/screens/CrashReportingScreen.tsx

```diff
@@ -46,6 +50,48 @@ export const CrashReportingScreen: React.FC = () => {
     }
   }
 
+  const styles = StyleSheet.create({
+    inputWrapper: {
+      padding: 4,
+      flex: 1,
+    },
+
+    formContainer: {
+      flexDirection: 'row',
+      alignItems: 'stretch',
+    },
+  });
+  const [userAttributeKey, setUserAttributeKey] = useState('');
+  const [userAttributeValue, setUserAttributeValue] = useState('');
+  const [crashNameValue, setCrashNameValue] = useState('');
+  const [crashFingerprint, setCrashFingerprint] = useState('');
+  const [crashLevelValue, setCrashLevelValue] = useState<NonFatalErrorType>(
+    NonFatalErrorType.error,
+  );
+
+  function sendCrash() {
+    try {
+      const error = Error(crashNameValue);
+      if (!error.message) {
+        const appName = 'Instabug Test App';
+        error.message = `Handled ${crashNameValue} From ${appName}`;
+      }
+      throw error;
+    } catch (err) {
```

Comment:
We are already passing `crashNameValue` to `new Error` as a message (the first argument to the `Error` constructor is the message), so `error.message` will be empty only when `crashNameValue` is empty. I think we can just put the message we want to put in the constructor, no need for this check in my opinion.